### DISCUSSION TITLE
Custom header for specific KFImage elements?

### DISCUSSION THREAD

**Discussion by LilaQ** [onevcat/Kingfisher#2068] [General]
Hi guys,

I know about the way to set custom headers to KingfisherManager default options, but this sadly isn't enough for me, since I need to provide different headers for different requests in my SwiftUI project.

How can I provide different options / headers for my SwiftUI KFImages?

Thanks in advance!

**Comment by LilaQ** [comment]
Okay I found the .requestModifier function, no clue I how missed that all the time :D

**Comment by onevcat** [comment]
Glad you found that! Yes, it is the expected way to append headers to the request. :]
